drivers/storage/portworx/util/util.go

```diff
@@ -499,21 +473,36 @@ func SetupContextWithToken(ctx context.Context, cluster *corev1alpha1.StorageClu
 		return ctx, nil
 	}
 
-	pxAuthSecret, err := GetAdminSecret(ctx, cluster, k8sClient)
+	pxAdminToken, err := GetSecretValue(ctx, cluster, k8sClient, "px-admin-token", "auth-token")
```

Comment:
nit: Can we use the constants for px-admin-token and auth-token?

Reply:
Will address in a future PR